go/registry/api/runtime.go

```diff
@@ -267,7 +292,7 @@ func (s *RuntimeStakingParameters) ValidateBasic(runtimeKind RuntimeKind) error
 const (
 	// LatestRuntimeDescriptorVersion is the latest entity descriptor version that should be used
 	// for all new descriptors. Using earlier versions may be rejected.
-	LatestRuntimeDescriptorVersion = 1
+	LatestRuntimeDescriptorVersion = 2
 
 	// Minimum and maximum descriptor versions that are allowed.
 	minRuntimeDescriptorVersion = 1
```

Comment:
When would we support version 1 runtime descriptors? Isn't the idea that fixgenesis will convert them and so we don't need to support version 1 here (so no need for version checks above and below)?

Reply:
Oops, this is a leftover from before I wrote the fixgenesis code, I'll change the minimum version to 2.